src/2_RTB/VerbaliInterni/VerbaleInterno_141123/content.typ

```diff
@@ -0,0 +1,61 @@
+#import "meta.typ": inizio_incontro, fine_incontro, luogo_incontro
+#import "functions.typ": glossary, team
+#let participants = csv("participants.csv")
+
+= Partecipanti
+/ Inizio incontro: #inizio_incontro
+/ Fine incontro: #fine_incontro
+/ Luogo incontro: #luogo_incontro
+
+#table(
+  columns: (3fr, 1fr),
+  [*Nome*], [*Durata presenza*],
+  ..participants.flatten()
+)
+= Sintesi Elaborazione Incontro
+
+/*************************************/
+/*    INSERIRE SOTTO IL CONTENUTO    */ 
+/*************************************/
+
+Nella riunione tenuta in data odierna sono stati trattati i seguenti punti, espressi nelle successive sezioni.
+
+== Chiarire conflitto riguardante le versioni del software e della documentazione
+È stato deciso di consultare i professori al fine di trovare una best practice per poter definire come la versione della documentazione e del prodotto software possano progredire in sincronia.
+
+== Elementi da aggiungere al documento "Norme di progetto"
+È stato fatto notare che i seguenti punti devono essere aggiunti al documento "Norme di progetto":
+- Va aggiunta una procedura per definire come le issues possano essere aggiunte in maniera asincrona, al _backlog_, da qualsiasi componente, nell'eventualità in cui si verifichino problemi o necessità da affrontare;
```

Comment:
togliere "va aggiunta" in quanto già scritto sopra

Reply:
Modificato come richiesto.